app/services/foreman_templates/uninstaller.rb

```diff
@@ -0,0 +1,13 @@
+module ForemanTempaltes
+  class Cleaner
```

Comment:
good catch, I did the last second rename since I realized we're not uninstalling the plugin :-)